typescript/web/src/components/metabase/metabase-dashboard.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import { Flex } from "@chakra-ui/react";
+import { chakraDecorator } from "../../utils/stories";
+import { KpiDashboard } from "./kpi-dashboard";
+
+export default {
+  title: "web/Metabase Dashboard",
```

Comment:
As the PR has been rebased on main. This file must now use the `storybookTitle` function to define the story title.

```suggestion
  title: storybookTitle(MetabaseDashboard),
```